src/applications/healthcare/questionnaire/containers/IntroductionPage.jsx

```diff
@@ -54,133 +47,89 @@ const IntroductionPage = props => {
-        </p>
-        <p>
-          <span className="vads-u-font-weight--bold">
-            If you don’t have an emergency, but you’re not sure what type of
-            care care you need,
-          </span>{' '}
-          call your nearest VA health facility.
-        </p>
+  const title = 'Primary care questionnaire';
+  const subTitle = facilityName;
+  return (
+    <div className="schemaform-intro healthcare-experience">
+      <FormTitle title={title} subTitle={subTitle} />
+      <h2 className="better-prepare-yours">
+        To help us better prepare for your appointment, fill out this short
+        questionnaire about your upcoming visit. This is where you can give us
+        more detail about your appointment or let us know if there are any
+        specific health issues you want to discuss with your provider.
+      </h2>
+      <section className="after-details">
+        <h3>What happens after I answer the questions?</h3>
         <p>
-          <a href="/find-locations/">Find your nearest VA health facility</a>
+          After you answer the questions, your questionnaire will be securely
+          sent to your provider for review. The questionnaire will also be added
+          to your medical record. Please try to submit the questionnaire [X]
+          days before your appointment.
         </p>
         <p>
-          <span className="vads-u-font-weight--bold">
-            If you need to talk to someone right away,
-          </span>{' '}
-          contact the Veterans Crisis Line. Whatever you’re struggling
-          with—chronic pain, anxiety, depression, trouble sleeping, anger, or
-          even homelessness—we can support you. Out Veterans Crisis Line is
-          confidential (private), free, and available 24/7.
+          <strong>
+            Your provider will discuss the information on your questionnaire
+            during your appointment.
+          </strong>
         </p>
-        <section>
+        <section className="emergency-call-out">
           <header>
-            To connect with a Veterans Crisis Line responder anytime, day or
-            night:
+            Note: If you need to talk to someone right away or need emergency
+            care,
           </header>
           <ul>
             <li>
-              Call the Veterans Crisis Line at{' '}
-              <Telephone contact={CONTACTS.CRISIS_LINE} /> and press 1, or
+              Call <Telephone contact="911" />, or
             </li>
             <li>
-              Start a{' '}
-              <a href="https://www.veteranscrisisline.net/ChatTermsOfService.aspx?account=Veterans%20Chat/">
-                confidential Veteran chat
-              </a>{' '}
-              , or
-            </li>
-            <li>
-              Text <a href="sms:838255">838255</a>.
+              Call the Veterans Crisis hotline at{' '}
+              <Telephone contact="800-273-8255" /> and select 1
```

Comment:
Could phone numbers be moved into a more common location? Just so that we don't have to touch it everywhere if the number changes for some reason in the future. Don't have to address it here necessarily but something to keep in mind for future refactors.

Reply:
Yes, that is on my radar to refactor the phone numbers.